Dockerfile

```diff
@@ -1,19 +1,20 @@
 # Use the official Docker Hub Ubuntu 14.04 base image
 FROM ubuntu:16.04
 
-# Update the base image
-RUN apt-get update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
-
 # Install Timesketch dependencies
-RUN apt-get -y install python-pip python-dev libffi-dev python-psycopg2
+RUN apt-get update \
+  && apt-get -y install python-pip \
+                        python-dev \
+                        libffi-dev \
+                        python-psycopg2 \
+                        software-properties-common
 
 # Install Plaso
-RUN apt-get -y install software-properties-common
-RUN add-apt-repository ppa:gift/stable
-RUN apt-get update
-RUN apt-get -y install python-plaso
+RUN add-apt-repository ppa:gift/stable \
```

Comment:
I'll double check this, but it seems it is already included in the ubuntu base image/packages previously installed.